crates/miden-lib/src/note/well_known_note.rs

```diff
@@ -336,71 +336,73 @@ impl WellKnownNote {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-/// Returns the receiver account ID parsed from the provided P2ID note inputs.
+/// Returns the receiver account ID parsed from the provided P2ID note storage.
 ///
 /// # Errors
 ///
 /// Returns an error if:
-/// - the length of the provided note inputs array is not equal to the expected inputs number of the
-///   P2ID note.
-/// - first two elements of the note inputs array does not form the valid account ID.
-fn parse_p2id_inputs(note_inputs: &[Felt]) -> Result<AccountId, StaticAnalysisError> {
-    if note_inputs.len() != WellKnownNote::P2ID.num_expected_inputs() {
+/// - the provided note storage length is not equal to the expected inputs number of the P2ID note.
+/// - first two elements of the note storage is not a valid account ID.
+fn parse_p2id_storage(note_storage: &[Felt]) -> Result<AccountId, StaticAnalysisError> {
+    if note_storage.len() != WellKnownNote::P2ID.num_expected_inputs() {
         return Err(StaticAnalysisError::new(format!(
-            "P2ID note should have {} inputs, but {} was provided",
+            "P2ID note should have {} storage items, but {} was provided",
```

Comment:
Perhaps we should just use one term to describe the storage items, as we currently use both `storage items` and `storage values`.